web/themes/custom/vote_gov/templates/partial/registration-date-mail.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * partial
+ * Voter registration mail-in deadline
+ *
+ */
+#}
+
+{#postmarked#}
```

Comment:
See the online template comment. It might make sense to use the same method for this file.